### PR TITLE
✨ Frontend: Simpler credits workflow

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

In order to make the credits workflows as simple as possible:
- [x] The backend will create a Wallet per user upon registration, therefore the creation of Wallets via the frontend has been removed. Also, we only allow users to buy credits for their personal wallet
- [x] A Wallet will only have one Accountant -> remove the "Promote to Accountant" option
- [x] Only allow sharing Wallets/Credits with users. For now, no Organizations.

### CODE REVIEW

**Review by matusdrobuliak66 — COMMENTED** [review]
test

**Review by ignapas — COMMENTED** [review]
I see no wallet after creating a new account... I am probably missing something here
image

File: services/static-webserver/client/source/class/osparc/desktop/wallets/WalletsList.js [hunk 1]
```diff
@@ -109,7 +112,7 @@ qx.Class.define("osparc.desktop.wallets.WalletsList", {
     __getCreateWalletSection: function() {
       const createWalletBtn = new qx.ui.form.Button().set({
         appearance: "strong-button",
-        label: this.tr("New Wallet"),
+        label: this.tr("New Credit Account"),
```
[the remark this answers is not in the document]
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/wallets/WalletsList.js:115]: PO's decision, we call it Credits account to distinguish it from the User's account

**Review by pcrespov — COMMENTED** [review]
:+1:

**Review by ignapas — APPROVED** [review]
I agree "credit account" sound like the bank's credit account... something that would charge you end of the month with some interest

**Review by pcrespov — APPROVED** [review]

**Pull request merged**